### PR TITLE
Feature/firesquid tips

### PR DESCRIPTION
**Pull request by jdomingos**
I've indexed locally and compare counters and output from existing prod one and this. All good!

### CODE REVIEW

File: prawns/tips/src/handlers/typeGetters/getTipsTipCall.ts [hunk 1]
```diff
@@ -42,8 +32,6 @@ export function getTipsTipCall(
       if (call.isV28) {
         return call.asV28;
       }
-
-      return call.asLatest;
     }
```
Comment (markdavison) [inline prawns/tips/src/handlers/typeGetters/getTipsTipCall.ts:42]: I think we either should have an exception in the place of the return call.asLatest or we should change the fallback exception message.

File: prawns/tips/src/handlers/tips.tip.extrinsic.ts [hunk 2]
```diff
@@ -1,29 +1,35 @@
...
-    const account = ctx.extrinsic.signer;
-    const rootAccount = decodeAddress(account);
-    const date = new Date(ctx.block.timestamp);
 
     const tipModel = await ctx.store.get(SubstrateTip, u8aToHex(tipCall.hash));
     if (!tipModel) {
       // NOTE: We don't break here because there can be old tippers or tips that are created through the treasury.NewTip.
       // For now we don't need to index those events.
-      console.log(`tips.tip.extrinsic::Tip not found: ${u8aToHex(tipCall.hash)}`);
+      console.log(
```
Comment (markdavison) [inline prawns/tips/src/handlers/tips.tip.extrinsic.ts]: I saw we have moved to use ctx.log for one other log. I think we should be consistent of either console or use their logger.

File: prawns/tips/src/handlers/typeGetters/getTipsTipCall.ts (hunk 1 above)
Comment (jdomingos) [reply prawns/tips/src/handlers/typeGetters/getTipsTipCall.ts:42]: We do have it, it's the `default`. Do you think I should make the code more readable?

Screenshot 2022-08-16 at 09 01 21

Comment (jdomingos) [reply prawns/tips/src/handlers/typeGetters/getTipsTipCall.ts:42]: Or the error more explicit :D

Comment (markdavison) [reply prawns/tips/src/handlers/typeGetters/getTipsTipCall.ts:42]: Sorry that is what I meant by fallback exception message (bad wording on my part). That exception should mean we don't have a type getter for that network. We should either make it more generic... or probably preferably just add an 'unexpected version' error. Should we create a new error type that we use for just this purpose everywhere? Then we can alert on it in future?

**Review by markdavison — APPROVED** [review]

**Pull request merged**